smoke-tests/smoke-tests.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
   implementation deps.slf4j
   implementation deps.opentelemetryProto
   implementation deps.opentelemetryApi
-  implementation "org.testcontainers:testcontainers:1.14.3"
+  implementation "org.testcontainers:testcontainers:1.15.0-rc2"
```

Comment:
Can't we use `deps.testcontainers` here?

Reply:
Indeed we can